spec/dummy/app/avo/resources/project.rb

```diff
@@ -22,7 +22,11 @@ def fields
       filterable: true,
       summarizable: true
     field :name, as: :text, required: true, sortable: true, default: "New project default name"
-    field :progress, as: :progress_bar, value_suffix: "%", display_value: true
+    field :progress,
+      as: :progress_bar,
+      value_suffix: "%",
+      display_value: true,
+      visible: -> { resource.view.form? || resource.record.progress&.positive? }
```

Comment:
```suggestion
      visible: -> do 
        # conditionally hiding the fields we can text that it's not going to break the table layout
        resource.view.form? || resource.record.progress&.positive?
      end
```

Reply:
@adrianthedev did you mean "we can test" instead "we can text"?

Reply:
yes 🤦‍♂️